src/app/static/src/app/components/plots/Choropleth.vue

```diff
@@ -59,12 +59,9 @@
             currentFeatures: function () {
                 return this.featuresByLevel[this.detail || 1]
             },
-            getColor: function () {
+            selectedColorFunction: function () {
                 return this.colorFunctions[this.indicator];
             },
```

Comment:
I took out showLegend, which was being used to hide the legend if min and max were the same, since values which were different but similar were being rounded to the same values in the legend. Anyway, this should not be an issue once we're getting sensible ranges from the API.